src/confluent_kafka/schema_registry/common/schema_registry_client.py

```diff
@@ -658,6 +658,17 @@ def from_dict(cls: Type[T], src_dict: Dict[str, Any]) -> T:
 
         return metadata
 
+@_attrs_define(frozen=True)
+class SchemaVersion:
+    subject: Optional[str]
+    version: Optional[int]
+
+    @classmethod
+    def from_dict(cls: Type[T], src_dict: Dict[str, Any]) -> T:
+        d = src_dict.copy()
```

Comment:
Kinda unnecessary, this is a shallow copy action that we immediately throw away. Could just make this a one-liner of `return cls(subject=src_dict.get('subject'), version=src_dict.get('version'))`